snapcraft/internal/meta/slots.py

```diff
@@ -55,16 +55,16 @@ def validate(self) -> None:
     def from_dict(cls, *, slot_dict: Dict[str, Any], slot_name: str) -> "Slot":
         """Return applicable Slot instance from dictionary properties."""
 
+        iface = slot_dict["interface"] if "interface" in slot_dict else slot_name
```

Comment:
I think the more idiomatic way to do this would be to just update the default value for `.get()` like so:
`interface = slot_dict.get("interface", slot_name)`

Reply:
You are right.